runtime/etc/src/mli/common_code.c

```diff
@@ -251,16 +251,16 @@ char * chpl_mli_connection_info(void* socket) {
   traveler++;
 
   int lenConnBoilerplate = strlen("tcp://:");
-  char* chpl_launch_masterip = getenv("CHPL_LAUNCH_MASTERIP");
-  chpl_mli_debugf("got env var value for CHPL_LAUNCH_MASTERIP: %s\n",
-                  chpl_launch_masterip);
+  char* chpl_rt_masterip = getenv("CHPL_RT_MASTERIP");
```

Comment:
You should be able to use `chpl_env_rt_get("MASTERIP", NULL)` now. There won't be any real benefit from that today, but if we ever add hooks to track what chapel env vars are set it's helpful.

Reply:
Since this code maybe be used without the rest of the Chapel runtime (depending on if it is called from the client side or from the server side), I'm going to leave this one alone but change the one in comm-gasnet-launch.c

Reply:
Alternatively, I could surround it in `ifdef`s since there are situations where it would get linked to the rest of the Chapel runtime.  Do you have a preference?

Reply:
The `chpl_env_rt_get()` function should be callable in the launcher.

Reply:
It might have been missed in the back-and-forth, but `chpl_env_rt_get()` is available in both the runtime and the launcher.  Is the problem (requiring the `#ifdef`) that this code will sometimes be used when not linked with _either_ of those?

Reply:
You were right :) I've tested the update, just didn't commit and push it yet

Reply:
And testing it on linux showed problems.  I'm going to back out that change, sorry!

Reply:
Can you say more about what the problems were?

Reply:
Duplicate definitions of symbols, like `chpl_library_finalize` (due to the client side for multilocale interoperability having its own definition that triggers our intentional set up, and chpl-env.c #include'ing chplrt.h).